my-app/src/store/reducers/columnReducer.ts

```diff
@@ -39,6 +39,22 @@ export const columnReducer = (state: IColumn = initialState.column, action: Acti
               columnList: state.columnList.filter((column) => column.columnId !== action.payload),
             };
           }
+          case Actions.SORT_COLUMN_LIST: {
+            console.log(action.payload)
+            const sortedList = state.columnList.map((c: ColumnItemType) => {
```

Comment:
1-letter variable name is normally not a good variable name. Make it, for instance, `column` instead of `c`